samples/client/petstore/php/SwaggerClient-php/lib/Model/MixedPropertiesAndAdditionalPropertiesClass.php

```diff
@@ -181,6 +181,11 @@ public function getUuid()
      */
     public function setUuid($uuid)
     {
+        if (is_null($uuid)) {
+            $this->container['uuid'] = $uuid;
+
+            return $this;
+        }
         $this->container['uuid'] = $uuid;
```

Comment:
@wing328 I saw it and we are using the latest master, but the problem is that the change you say needs to be done also in the setter.